docs-source/content/samples/simple/azure-kubernetes-service/_index.md

```diff
@@ -5,1017 +5,45 @@ weight: 8
 description: "Sample for using the operator to set up a WLS cluster on the Azure Kubernetes Service."
 ---
 
-This sample demonstrates how to use the [Oracle WebLogic Server Kubernetes Operator](/weblogic-kubernetes-operator/) (hereafter "the operator") to set up a WebLogic Server (WLS) cluster on the Azure Kubernetes Service (AKS). After going through the steps, your WLS domain runs on an AKS cluster instance and you can manage your WLS domain by accessing the WebLogic Server Administration Console.
 
-AKS is a managed Kubernetes Service that lets you quickly deploy and manage Kubernetes clusters. To learn more, please see the [Azure Kubernetes Service (AKS)](https://docs.microsoft.com/azure/aks/) overview page.
+### Contents
 
-#### Contents
+   - [Introduction](#introduction)
+     - [Azure Kubernetes Service cluster](#azure-kubernetes-service-cluster)
+     - [Domain home source types](#domain-home-source-types)
+   - [Domain in PV]({{< relref "/samples/simple/azure-kubernetes-service/domain-on-pv.md" >}}): Running the WebLogic cluster on AKS with domain home on PV
+   - [Model in Image]({{< relref "/samples/simple/azure-kubernetes-service/model-in-image.md" >}}): Running the WebLogic cluster on AKS with domain model in image
+   - [Troubleshooting]({{< relref "/samples/simple/azure-kubernetes-service/troubleshooting.md" >}})
 
- - [Prerequisites](#prerequisites)
- - [Create the AKS cluster](#create-the-aks-cluster)
- - [Create storage and set up file share](#create-storage-and-set-up-file-share)
- - [Install WebLogic Server Kubernetes Operator](#install-weblogic-server-kubernetes-operator)
- - [Create WebLogic domain](#create-weblogic-domain)
- - [Automation](#automation)
- - [Deploy sample application](#deploy-sample-application)
- - [Access WebLogic Server logs](#access-weblogic-server-logs)
- - [Troubleshooting](#troubleshooting)
- - [Useful links](#useful-links)
 
-#### Prerequisites
+### Introduction
 
-This sample assumes the following prerequisite environment setup:
+This sample demonstrates how to use the [Oracle WebLogic Server Kubernetes Operator]({{< relref "/" >}}) (hereafter "the operator") to set up a WebLogic Server (WLS) cluster on the Azure Kubernetes Service (AKS). After going through the steps, your WLS domain runs on an AKS cluster.  You have several options for managing the cluster, depending on which [domain home source type]({{< relref "/userguide/managing-domains/choosing-a-model/" >}}) you choose.  With Domain in PV, you can manage your WLS domain by accessing the WebLogic Server Administration Console or WLST.  With Model in Image you use the operator to perform WLS administrative operations.
```

Comment:
The 'choosing-a-model' relref didn't work in my version of hugo:

[tbarnes-1 docs-source]$ ./runpublic.sh
Building sites … ERROR 2021/02/24 17:13:24 [en] REF_NOT_FOUND: Ref "/userguide/managing-domains/choosing-a-model/" from page "samples/simple/azure-kubernetes-service/_index.md": page not found
WARN 2021/02/24 17:13:24 .File.UniqueID on zero object. Wrap it in if or with: {{ with .File }}{{ .UniqueID }}{{ end }}

Please append "/_index.md" to the reference.

Reply:
@rosemarymarano FYI - I don't know if this is necessary, but I don't see how it can hurt.